### DISCUSSION TITLE
My Country Isn’t on the List, I Am Unable to Set up Two Factor Authentication

### DISCUSSION THREAD

**Discussion by devisasari** [community/community#36157] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to set up two-factor authentication for my Github account. However, as I am a resident of Turkey, I am unable to do it. I am unable to download the Github app to my phone. Will Turkey get access to this feature? Why not yet? What can I do until Turkey gets this feature?

**Comment by kaplanark** [comment, marked as answer]
You can currently use Google Authenticator as a fix. You should get in touch with Github if you have any other inquiries; upon request, they will respond.
